plugins/search/providers/elastic_search/lib/model/search/kESearchQueryManager.php

```diff
@@ -63,13 +64,26 @@ public static function getPartialQuery($searchItem, $fieldName, &$queryAttribute
 		$multiMatchFieldBoostFactor = self::MATCH_FIELD_BOOST_FACTOR * $fieldBoostFactor;
 		$matchQuery->setBoostFactor($multiMatchFieldBoostFactor);
 		$matchQuery->setAnalyzer(self::KALTURA_TEXT_PARTIAL_SEARCH_ANALYZER);
+		$shouldReduceResults = self::isPartnerShouldReduceResults(kBaseElasticEntitlement::$partnerId);
+		if ($shouldReduceResults)
+		{
+			$matchQuery->setOperator(self::OP_AND);
+			$cuttOffFreq = kConf::get('cutoff_frequency','elasticDynamicMap');
```

Comment:
1. Add const to map name - elasticDynamicMap
2. Add const to cutoff_frequency
3. create function getCutoffFreq() and move the call inside

Reply:
Add default value to the kconf call